### ISSUE TITLE
Printing out the validation loss for classification

### ISSUE THREAD

**Issue by f-izzat** [ultralytics/ultralytics#2024]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

So currently training logs look like this, with `val=True`
```
Epoch    GPU_mem       loss  Instances       Size
      1/100         0G     0.3482         16        224: 100%|██████████| 90/90 [03:44<00:00,  2.49s/it]
               classes   top1_acc   top5_acc: 100%|██████████| 15/15 [02:14<00:00,  8.99s/it]
                   all      0.254          1
```
Which is great but I was wondering where the validation loss was for each epoch and found out that its logged into `results.csv` is there any way to print this out in terminal?.
<img width="911" alt="image" src="https://user-images.githubusercontent.com/97789955/232029735-793c1cea-bfe4-4f18-886c-f2a03090f203.png">

Thanks

### Use case

To know if the model is over/under fitting. Pretty standard metric to print

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
Thanks for the feedback. Yes, currently it only logs accuracies on the console. We'll fix it.
@glenn-jocher @Laughing-q You think this should be on the TODOs?

**Comment by Laughing-q** [comment]
@AyushExel ah but if we're going to add val/loss of classify in terminal log then we should also implement this feature to detect/segment/pose tasks, but I don't think the terminal has more space to show val/loss for detect/segment/pose tasks.😂
here's an example log of training segmentation which has already filled my terminal.
![image](https://user-images.githubusercontent.com/61612323/232232318-3681eccb-bb3c-4397-8e3f-7bdc8e1e76f9.png)

@f-izzat I think we can also know if the model is over/under fitting by `top1_acc` and `top5_acc` right?

**Comment by f-izzat** [comment]
@Laughing-q Are the top-n metrics calculated on the training set? For example here, by the 12th epoch the training metrics seem to do okay but the validation is still above 1 (I guess its relative to my problem at hand).
<img width="564" alt="image" src="https://user-images.githubusercontent.com/97789955/232232786-da311748-b097-452e-b5e9-187883a99659.png">

It would however still be great to see the progression in validation, would it be possible to instead chose which metrics to print rather than all? For example I have 4 classes, which means that the top-5 being 1 wouldnt really be useful to me

**Comment by Laughing-q** [comment]
@f-izzat top-n is calculated on val set.
